Makefile

```diff
@@ -28,7 +32,16 @@ format: ## Format the code
 
 # Inspired by https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
 .PHONY: help
+help:
```

Comment:
Code starting from 35 and until 43 should not be there. This part was replaced by line 44.